src/assets/scripts/client/UiController.js

```diff
@@ -1,595 +1,592 @@
-
-        this.$switchAirport.addClass(SELECTORS.CLASSNAMES.ACTIVE);
-    }
-
-    /**
-     * @for UiController
-     * @method _onClickCloseAirportDialog
-     * @private
-     */
-    _onClickCloseAirportDialog() {
-        this.$airportDialog.removeClass(SELECTORS.CLASSNAMES.OPEN);
-        this.$switchAirport.removeClass(SELECTORS.CLASSNAMES.ACTIVE);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleAirportList
-     */
-    onToggleAirportList() {
-        if (this.$airportDialog.hasClass(SELECTORS.CLASSNAMES.OPEN)) {
-            this._onClickCloseAirportDialog();
-
-            return;
-        }
-
-        this._onClickOpenAirportDialog();
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleLabels
-     * @param {jquery event}
-     */
-    onToggleLabels(event) {
-        $(event.target).closest(SELECTORS.DOM_SELECTORS.CONTROL).toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
-
-        this._eventBus.trigger(EVENT.TOGGLE_LABELS);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleRestrictedAreas
-     */
-    onToggleRestrictedAreas(event) {
-        $(event.target).closest(SELECTORS.DOM_SELECTORS.CONTROL)
-            .toggleClass(`${SELECTORS.DOM_SELECTORS.WARNING_BUTTON} ${SELECTORS.CLASSNAMES.ACTIVE}`);
-
-        this._eventBus.trigger(EVENT.TOGGLE_RESTRICTED_AREAS);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleSids
-     * @param event {jquery event}
-     */
-    onToggleSids(event) {
-        $(event.target).closest(SELECTORS.DOM_SELECTORS.CONTROL).toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
-
-        this._eventBus.trigger(EVENT.TOGGLE_SID_MAP);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleStars
-     * @param event {jquery event}
-     */
-    onToggleStars(event) {
-        $(event.target).closest(SELECTORS.DOM_SELECTORS.CONTROL).toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
-
-        this._eventBus.trigger(EVENT.TOGGLE_STAR_MAP);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleTerrain
-     * @param event {jquery event}
-     */
-    onToggleTerrain(event) {
-        $(event.target).closest(SELECTORS.DOM_SELECTORS.CONTROL).toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
-
-        this._eventBus.trigger(EVENT.TOGGLE_TERRAIN);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleVideoMap
-     * @param event {jquery event}
-     */
-    onToggleVideoMap(event) {
-        $(event.target).closest(SELECTORS.DOM_SELECTORS.CONTROL).toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
-
-        this._eventBus.trigger(EVENT.TOGGLE_VIDEO_MAP);
-    }
-
-    /**
-     * @for UiController
-     * @method onToggleOptions
-     */
-    onToggleOptions() {
-        this.$toggleOptions.toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
-        this.$optionsDialog.toggleClass(SELECTORS.CLASSNAMES.OPEN);
-    }
-}
-
-export default new UiController();
+import $ from 'jquery';

```

Comment:
Why has this entire file changed?

Reply:
The file uses Windows line endings now.

@JPmAn24 Please use Unix line endings.